lib/parse.js

```diff
@@ -2236,17 +2236,86 @@ function parse($TEXT, options) {
         })
     }
 
+    function import_nameAsterisk() {
+        var start = S.token;
+        var foreign_name;
+        var name;
+
+        next();
+
+        var end = prev();
+
+        name = new AST_Symbol({
```

Comment:
(GitHub isn't showing notifications properly to me for the past two days.)

A symbol is fine here, though it feels strange at a glance why `AST_SymbolImport` would appear in an `export` statement.

@avdg any better ideas?